src/intervals/interval_operations/pointwise_boolean.jl

```diff
@@ -0,0 +1,206 @@
+>=(::PointwisePolitic{:ternary}, x::Interval, y::Interval) = !<(PointwisePolitic{:ternary}(), x, y)
+
+# Boolean functions
+# NOTE this interacts with flavors.
+function isinf(::PointwisePolitic{:ternary}, x::Interval)
+    if contains_infinity(x)
+        isthing(x) && return true
+        return missing
+    end
+
+    return false
+end
+
+isfinite(::PointwisePolitic{:ternary}, x::Interval) = !isinf(PointwisePolitic{:ternary}(), x)
+iszero(::PointwisePolitic{:ternary}, x::Interval) = ==(PointwisePolitic{:ternary}(), x, 0)
+
+function isinteger(::PointwisePolitic{:ternary}, x::Interval)
+    (x.lo == x.hi) && isinteger(x.lo) && return true
+    floor(x.hi) < ceil(x.lo) && return false
+    return missing
+end
+
+
+## :interval
+"""
+    BooleanInterval
+
+Type representing a set containing `true` and/or `false`.
+
+Test what it contains using `in`. For example `true in BooleanInterval(true, false)`.
+"""
+struct BooleanInterval
+    has_true::Bool
+    has_false::Bool
+
+    function BooleanInterval(a::Bool, b::Bool)
+        a == b && throw(ArgumentError("boolean interval with content [$a, $b] doesn't make sense."))
+        return new(true, true)
+    end
+
+    function BooleanInterval(a::Bool)
+        a && return new(true, false)
+        return new(false, true)
+    end
+
+    BooleanInterval(::Missing) = new(true, true)
+end
+
+in(bool::Bool, bi::BooleanInterval) = bool ? bi.has_true : bi.has_false
+
+function show(io::IO, bi::BooleanInterval)
+    true in bi && false in bi && return print(io, "[true, false]")
+    true in bi && return print(io, "[true]")
+    false in bi && return print(io, "[false]")
+end
+
+for op in pointwise_bool_operations
+    @eval function $op(::PointwisePolitic{:interval}, x::Interval, y::Interval)
+        return BooleanInterval($op(PointwisePolitic{:ternary}(), x, y))
+    end
+end
+
+for f in pointwise_bool_functions
+    @eval function $f(::PointwisePolitic{:interval}, x::Interval)
+        return BooleanInterval($f(PointwisePolitic{:ternary}(), x))
+    end
+end
+
+
+## :is_all
+for op in pointwise_bool_operations
+    @eval function $op(::PointwisePolitic{:is_all}, x::Interval, y::Interval)
+        ternary_res = $op(PointwisePolitic{:ternary}(), x, y)
+        ismissing(ternary_res) && return false
+        return ternary_res
+    end
+end
+
+for f in pointwise_bool_functions
+    @eval function $f(::PointwisePolitic{:is_all}, x::Interval)
+        ternary_res = $f(PointwisePolitic{:ternary}(), x)
+        ismissing(ternary_res) && return false
+        return ternary_res
+    end
+end
+
+
+## Number-interval comparisons
+for op in pointwise_bool_operations
+    @eval function $op(P::PointwisePolitic, x::F, y::Real) where {F<:Interval}
+        return $op(P, x, F(y))
+    end
+
+    @eval function $op(P::PointwisePolitic, x::Real, y::F) where {F<:Interval}
+        return $op(P, F(x), y)
+    end
+end
+
+
+## Default behaviors
+pointwise_politic() = PointwisePolitic{:ternary}()
```

Comment:
I think the default behavior should be the standard-compliant

Reply:
Could you point me to where it is defined in the standard?